src/main/java/ch/njol/skript/expressions/ExprTarget.java

```diff
@@ -32,12 +16,29 @@
 import ch.njol.skript.registrations.EventValues;
 import ch.njol.util.Kleenean;
 import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.GameMode;
+import org.bukkit.Location;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.entity.Mob;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.bukkit.event.entity.EntityTargetEvent;
+import org.bukkit.util.RayTraceResult;
+import org.bukkit.util.Vector;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.function.Predicate;
 
 @Name("Target")
 @Description({
 	"For players this is the entity at the crosshair.",
 	"For mobs and experience orbs this is the entity they are attacking/following (if any).",
-	"Display entities have a hit box of 0, so you should use 'target display' to collect Display entities",
+	"The 'ray size' and 'ignoring blocks' options are only valid for players' targets.",
+	"The 'ray size' option effectively increases the area around the crosshair an entity can be in. It does so " +
+	"by expanding the hitboxes of entities by the given amount. Display entities have a hit box of 0, " +
+	"so using the 'ray size' option can be helpful when targeting them.",
```

Comment:
```suggestion
	"The 'ray size' option effectively increases the area around the crosshair an entity can be in. It does so "
		+ "by expanding the hitboxes of entities by the given amount. Display entities have a hit box of 0, "
		+ "so using the 'ray size' option can be helpful when targeting them.",
```
Don't have to, I'm just used to it after Kenzie had suggested it to me before,

Reply:
I am against this for string concatenation operations but not other operations simply due to the practice of writing line breaks in text always using the - character at the end of a line, not the start of a new one. kenzie and i have gone back and forth on this and i cannot recall if there was compromise or not